modin/experimental/engines/omnisci_on_ray/io.py

```diff
@@ -140,7 +205,11 @@ def read_csv(
         try:
             if eng in ["pandas", "c"]:
                 return cls._read(**mykwargs)
-
+            use_modin_impl, error_message = cls._read_csv_check_support(
+                mykwargs,
+            )
+            if not use_modin_impl:
+                raise ArrowEngineException(error_message)
```

Comment:
what is the point of _returning_ an error message to raise an exception? isn't it easier to just raise the exception in `_read_csv_check_support()` directly?

Reply:
This is done to maintain the common API of "check support" functions([case1](https://github.com/modin-project/modin/blob/master/modin/engines/base/io/text/csv_dispatcher.py#L176), [case2](https://github.com/modin-project/modin/blob/master/modin/engines/ray/generic/io.py#L52)). Does this make sense?

Reply:
Okay, let's make a "cleanup issue" to not clutter this PR.

Reply:
Ok, issue was created https://github.com/modin-project/modin/issues/3166.